### ISSUE TITLE
Check user journey maps

### ISSUE THREAD

**Issue by christianbrb** [raiden-network/light-client#310]
## Description
@roy-castrillo has documented the new user flow. We need to go through it and check if processes make sense to everyone within our team.

- Manage Channels
- Make Payments

## Acceptance criteria
- User journey is comparable to the process in the existing UI – they make sense together. 
- All team members understand both user flows and agree on their current status.

## Tasks
- [ ] Team discuses user flows during refinement session
- [ ] Edits are made and final flows are shared internally

**Comment by roy-castrillo** [comment]
When I briefed in @taleldayekh on the current stand of the Light Client, he came out with critical observations that question the current user journey. He also made wireframes as fix/improvement proposals that we've been discussing the last week.

During the refinement call on Friday 11.10.2019, the whole team agreed to treat the Light Client as a showcase of all features offered by the SDK. We defined two important principles for the future of the dApp: 1) We build a conceptual framework where we adapt usability to enable all possible features, so that users can experience the full potential of the SDK 2) We go from complex to simple, as doing it the other way around would require more resources on the back-end. 

With this in mind, Talel and I have reworked the user journey maps. The main map is 'Making a transfer', which provides an overview of all main steps that users need to follow from the moment they face the 'Connect Wizard' screen all the way to the competition of a transfer.

![making transfer](https://user-images.githubusercontent.com/47739420/66831831-dc147700-ef58-11e9-9dff-caed644864c4.jpg)
